backend/routing/QuizRoutes.go

```diff
@@ -3,22 +3,35 @@ package routing
 import (
 	"encoding/json"
 	"net/http"
-
-	"github.com/fourtf/studyhub/utils"
 )
 
 type apiResponse struct {
-	Error string      `json:"error"`
-	Data  interface{} `json:"data"`
+	Data interface{} `json:"data"`
 }
 
-type quiz struct {
+const (
+	multipleCoiceKey = "multiple-coice"
+	freeTextKey      = "free-text"
+)
```

Comment:
In Go you can create an enum like this:
```suggestion
type QuestionType string
const (
      multipleChoice QuestionType = "multiple-coice"
	freeText      = "free-text"
)
```
I realized that you don't need this when you execute the following change as we can use an instanceof-check in the frontend